uiowa/uiowa
# Facilities' alerts template update

# To Test

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local uli node/3501
```

- Observe no import errors and a single column template in the same order as https://github.com/uiowa/uiowa/issues/6973#issuecomment-1749295647
- See that the dispatch template (https://github.com/uiowa/uiowa/issues/6973#issuecomment-1765101546 OR https://apps.its.uiowa.edu/dispatch/communications/1238605612/edit) is the same as https://github.com/uiowa/uiowa/issues/6973#issuecomment-1749295647

<img width="455" alt="Screenshot 2023-10-17 at 3 20 27 PM" src="https://github.com/uiowa/uiowa/assets/4663676/094fcc29-37b4-4cdb-a7a5-dd72ea9bbf57">
